doc/source/examples/multiphysics/dam-break-VOF/dam-break-VOF.rst

```diff
@@ -0,0 +1,255 @@
+    time-steps are modified during the simulation to keep the max CFD 
+    below a given threshold.
+
+.. code-block:: text
+
+    # --------------------------------------------------
+    # Simulation Control
+    #---------------------------------------------------
+    subsection simulation control
+        set method                         = bdf1
+        set time end                       = 4.1
+        set time step                      = 0.01
+        set adapt                          = true
+        set max cfl                        = 0.5
+        set stop tolerance                 = 1e-5
+        set adaptative time step scaling   = 1.1
+        set number mesh adapt              = 0
+        set output name                    = dam-break_VOF
+        set output frequency               = 20
+        set output path                    = ./Output/
+        set subdivision                    = 1      
+    end
+
+Make sure to create a directory named `Output` in the same directory 
+you are calling the solver from.   
+
+The ``multiphysics`` subsection enables to turn on `(true)` 
+and off `(false)`` the physics of interest. Here ``VOF`` and 
+``interface sharpening`` are chosen.
+
+.. note:: 
+     If the interface sharpening is not selected, the interface 
+     between phases will become blurry (due to diffusion). 
+     The following section defines the interface sharpening 
+     parameters.
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Multiphysics
+    #---------------------------------------------------
+    subsection multiphysics
+        set VOF = true
+        set interface sharpening = true
+    end 
+
+In the ``interface sharpening`` subsection, the parameters required for 
+sharpening the interface are defined. The current interface 
+sharpening method consists of two steps. These steps are explained as
+follows: 
+
+""""""""""""""""""""""""""
+1- Phase fraction limiter
+""""""""""""""""""""""""""
+
+.. math:: 
+    \phi := min \left( max \left(\phi^{old},0 \right),1 \right)
+
+""""""""""""""""""""""""
+2 -Interface sharpening 
+""""""""""""""""""""""""
+.. math::
+    \phi :=
+    \begin{cases}
+     c^{1-\alpha} \phi^{\alpha} &  (0 \leq \phi < c  ) \\
+     1-(c-1)^{1-\alpha}(1-\phi)^{\alpha} & (c \leq \phi \leq 1  ) 
+    \end{cases}
+
+
+where :math:`\phi`, :math:`c`, and :math:`\alpha` denote phase fraction, 
+sharpening threshold, and interface sharpness respectively. 
+This interface sharpening method was proposed by `Aliabadi and Tezduyar (2000)`_.  
+
+.. _Aliabadi and Tezduyar (2000):  https://www.sciencedirect.com/science/article/pii/S0045782500002000
+
+`Sharpening frequency` is an integer parameter that defines the 
+frequency of interface sharpening; sharpening threshold defines 
+a phase fraction threshold for interface sharpening (generally :math:`0.5`);
+and interface sharpness is a model parameter which is generally in
+the range of :math:`(1-2]`.
```

Comment:
```suggestion
``Sharpening frequency`` is an integer parameter that defines the 
frequency of the interface sharpening; ``sharpening threshold`` defines 
a phase fraction threshold for interface sharpening (generally :math:`0.5`);
and ``interface sharpness`` is a model parameter which is generally in
the range of :math:`(1-2]`.
```

Can you give guidelines on the interface sharpness parameter? What is a good default value for it.